### DISCUSSION TITLE
Notifikasi Order

### DISCUSSION THREAD

**Discussion by mediasaranainformatika** [kirimwa-id/forum#107] [Pamerkan]
Terimakasih mas Rio, integrasi KirimWA nya sudah berjalan dengan baik. Sangat membantu Admin Toko memproses orderan yang masuk.

![IMG_20220305_144203](https://user-images.githubusercontent.com/8298911/156871891-90f3ef74-be7c-45e8-8d8c-9b69e4c732e8.jpg)

**Comment by rioastamal** [comment]
@mediasaranainformatika Terima kasih telah menggunakan API KirimWA.id.
